Mapbox/MapboxMapsFoundation/Camera/CameraManager.swift

```diff
@@ -175,13 +175,21 @@ public class CameraManager {
             return
         }
 
-        guard mapView.cameraView.camera != newCamera else {
+        let clampedCamera = CameraOptions(
+            center: camera.center,
+            padding: camera.padding,
+            anchor: camera.anchor,
+            zoom: camera.zoom?.clamped(to: mapCameraOptions.minimumZoomLevel...mapCameraOptions.maximumZoomLevel),
+            bearing: optimizeBearing(startBearing: mapView.bearing, endBearing: camera.bearing),
+            pitch: camera.pitch?.clamped(to: mapCameraOptions.minimumPitch...mapCameraOptions.maximumPitch))
```

Comment:
Could we also increase the default `maximumPitch` value in `MapCameraOptions`?

Reply:
Yes. What should I change it to?

Reply:
Updated to 85. cc: @mapbox/maps-android